### PR TITLE
Add remove trash bin cover image to tip probe modal

### LINKED ISSUE #1030: Usability: Add Improved Trash Bin Guidance
As a user, I would like clear visual guidance on removing the trash bin before tip probe, so that I don't accidentally attempt tip probe with the tip probe box covered.

(note: users look for a 'cover', and when they don't find one, assume the trash bin is already removed)

## Acceptance Criteria
- Update pre-tip probe dialogue box with visual guidance on removing trash bin

## Design
- Zeplin -> Run App -> 'Remove Trash modal'

### LINKED ISSUE #1169: Run: change 'estimated run time' to 'run time
Run page in app says "estimated run time"

change to say "Run time:"

### PR DESCRIPTION
**Pull request by Kadee80**
## overview
This PR:
closes #1030 (Add Improved Trash Bin Guidance)
and as long as I was making UI changes ... closes #1169 (Run: change 'estimated run time' to 'run time)

screen shot 2018-04-12 at 3 01 13 pm
screen shot 2018-04-12 at 3 01 34 pm

## changelog

- Update copy and and image to match designs
- Change `AlertModal` flat buttons to outline buttons in comp_lib 
- Took the libery changing estimated run time -> run time in run panel

## review requests

try out tip probe, the prompt to remove trash bin cover now has updated copy and img
check out run screen for copy change
@IanLondon I touched the alert modal in comp_lib. please let me know if i messed up any of your styling in PD

### CODE REVIEW

File: app/src/components/ConfirmTipProbeModal/Contents.js [hunk 1]
```diff
@@ -2,22 +2,24 @@
 // contents of the ContinueTipProbeModal container
 import * as React from 'react'
 
+import removeTrashSrc from './trash@3x.png'
```
Comment (mcous) [inline app/src/components/ConfirmTipProbeModal/Contents.js]: I don't care, but we could also put this in `ConfirmTipProbeModal/images`

File: components/src/modals/modals.css [hunk 2]
```diff
@@ -39,4 +39,8 @@
 .alert_modal_buttons {
   float: right;
   margin-top: 1rem;
+
+  & button {
```
Comment (mcous) [inline components/src/modals/modals.css]: This won't work if one or more of the buttons use `Component: somethingNotAButton`. We should either:

1. Add a `className` to the buttons in `AlertModal` with this rule
2. Change the rule to something like `& *`

Personally, I prefer (1) but either are acceptable

File: app/src/components/ConfirmTipProbeModal/Contents.js (hunk 1 above)
Comment (Kadee80) [reply app/src/components/ConfirmTipProbeModal/Contents.js]: sure thing that was a lazy move

File: components/src/modals/modals.css (hunk 2 above)
Comment (Kadee80) [reply components/src/modals/modals.css]: 1 sounds better to me as well

**Pull request merged**